staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/register.go

```diff
@@ -37,8 +37,9 @@ func Resource(resource string) schema.GroupResource {
 }
 
 var (
-	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
-	AddToScheme   = SchemeBuilder.AddToScheme
+	SchemeBuilder    = runtime.NewSchemeBuilder(addKnownTypes)
+	AddToScheme      = SchemeBuilder.AddToScheme
+	AddToSchemeOrDie = SchemeBuilder.AddToSchemeOrDie
```

Comment:
would prefer some compositional or even local "or die" logic instead of defining it in every group.

Reply:
This can be a follow-up, with all the good this PR does.

Reply:
Addressing thsi now